### PR TITLE
Fixed Issue on UI, compare and login page

### PR DESCRIPTION
**Pull request by amit-webkul**

### CODE REVIEW

File: packages/Webkul/SocialLogin/src/Http/routes.php [hunk 1]
```diff
@@ -5,11 +5,11 @@
 Route::group(['middleware' => ['web', 'locale', 'theme', 'currency']], function () {
     Route::controller(LoginController::class)->prefix('customer/social-login/{provider}')->group(function () {
         Route::get('',  'redirectToProvider')->defaults('_config', [
```
Comment (jitendra-webkul) [inline packages/Webkul/SocialLogin/src/Http/routes.php:7]: We have removed route default parameter then why we are still using it here

File: packages/Webkul/Shop/src/Resources/views/components/products/card.blade.php [hunk 2]
```diff
@@ -221,14 +221,14 @@ class="bs-primary-button px-[30px] py-[10px] whitespace-nowrap"
 
                             localStorage.setItem('compare_items', JSON.stringify(items));
 
-                            this.$emitter.emit('add-flash', { type: 'success', message: '@lang('Item added successfully to compare list')' });
+                            this.$emitter.emit('add-flash', { type: 'success', message: "@lang('shop::app.components.products.card.add-to-compare')" });
```
Comment (devansh-webkul) [inline packages/Webkul/Shop/src/Resources/views/components/products/card.blade.php:224]: Whenever use qoutes then use double curly braces with trans method,

~~~php
'{{ trans('') }}'
~~~

File: packages/Webkul/Shop/src/Resources/views/products/view.blade.php [hunk 3]
```diff
@@ -362,14 +370,14 @@ class=" flex justify-center items-center gap-[10px] cursor-pointer"
 
                                 this.setStorageValue(this.getCompareItemsStorageKey(), existingItems);
 
-                                alert('Added product in compare.');
+                                this.$emitter.emit('add-flash', { type: 'success', message: "@lang('shop::app.products.add-to-compare')" });
```
Comment (devansh-webkul) [inline packages/Webkul/Shop/src/Resources/views/products/view.blade.php:373]: Same here.

File: packages/Webkul/Shop/src/Resources/views/products/view.blade.php [hunk 4]
```diff
@@ -362,14 +370,14 @@ class=" flex justify-center items-center gap-[10px] cursor-pointer"
...
                             } else {
-                                alert('Product is already added in compare.');
+                                this.$emitter.emit('add-flash', { type: 'warning', message: "@lang('shop::app.products.already-in-compare')" });
```
Comment (devansh-webkul) [inline packages/Webkul/Shop/src/Resources/views/products/view.blade.php:375]: Same here.

File: packages/Webkul/Shop/src/Resources/views/products/view.blade.php [hunk 5]
```diff
@@ -362,14 +370,14 @@ class=" flex justify-center items-center gap-[10px] cursor-pointer"
...
                             }
                         } else {
                             this.setStorageValue(this.getCompareItemsStorageKey(), [productId]);
 
-                            alert('Added product in compare.');
+                            this.$emitter.emit('add-flash', { type: 'success', message: "@lang('shop::app.products.add-to-compare')" });
```
Comment (devansh-webkul) [inline packages/Webkul/Shop/src/Resources/views/products/view.blade.php:380]: Same here.

**Review by devansh-webkul — APPROVED** [review]

**Review by jitendra-webkul — APPROVED** [review]

**Comment by devansh-webkul** [comment]
Conflicts.

**Pull request merged**